### DISCUSSION TITLE
Add other momentum distributions

### DISCUSSION THREAD

**Discussion by rlouf** [blackjax-devs/blackjax#94] [New features]
We currently only implement a normally-distributed moment distribution, but it could also be distributed according to a Laplace, Cauchy, etc distribution.

For that we would need to transform `gaussian_euclidean` to `euclidean` and pass the momentum distribution as an argument (default unit normal).

**Comment by rlouf** [comment]
See https://arxiv.org/abs/1706.02649

**Comment by junpenglao** [comment]
🤖 *This reply was drafted with Claude and reviewed by me (@junpenglao).*

Status: still gaussian-only (`gaussian_euclidean`, `gaussian_euclidean_low_rank`, `gaussian_riemannian` in `blackjax/mcmc/metrics.py`). Non-Gaussian momentum (Laplace/Cauchy/etc.) is not yet implemented — keeping this open. A PR generalizing `metrics.py` to accept a momentum distribution would be welcome.
